### PR TITLE
Add upgrade section for core-1 and test-core-1 chains

### PR DESCRIPTION
**Pull request by Anmol1696**

### CODE REVIEW

File: core-1/upgrades/v2/guide.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+* Bump [PersistenceCore](https://github.com/persistenceOne/persistenceCore) to [0.2.3](https://github.com/persistenceOne/persistenceCore/releases/tag/v0.2.3) which notably includes fixes for the vesting accounts and two new modules outlined below. For a detailed difference between PersistenceCore [v0.1.3](https://github.com/persistenceOne/persistenceCore/releases/tag/v0.1.3) and [v0.2.3](https://github.com/persistenceOne/persistenceCore/releases/tag/v0.2.3) check [diff between 0.1.3 and 0.2.3](https://github.com/persistenceOne/persistenceCore/compare/v0.1.3...v0.2.3).
+* Add the [authz](https://github.com/cosmos/cosmos-sdk/tree/v0.44.6/x/authz/spec) module to the Cosmos Hub. \`x/authz\` is an implementation of a Cosmos SDK module, per [ADR 30](https://github.com/cosmos/cosmos-sdk/blob/v0.44.6/docs/architecture/adr-030-authz-module.md), that allows granting arbitrary privileges from one account (the granter) to another account (the grantee). Authorizations must be granted for a particular Msg service method one by one using an implementation of the \`Authorization\` interface.
```
Comment (puneet2019) [inline core-1/upgrades/v2/guide.md:10]:
```suggestion
* Add the [authz](https://github.com/cosmos/cosmos-sdk/tree/v0.44.6/x/authz/spec) module to the Persistence mainnet. \`x/authz\` is an implementation of a Cosmos SDK module, per [ADR 30](https://github.com/cosmos/cosmos-sdk/blob/v0.44.6/docs/architecture/adr-030-authz-module.md), that allows granting arbitrary privileges from one account (the granter) to another account (the grantee). Authorizations must be granted for a particular Msg service method one by one using an implementation of the \`Authorization\` interface.
```

File: core-1/upgrades/v3/guide.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+When chain reaches the block height, the chain will pause with panic message followed by endless peer logs.
+Copy the `upgrade-info.json` to the correct location with
+```
+cp ~/data/upgrade-info.json ~/.persistenceCore/data/
+```
+
+Now cosmovisor will automatically catch the upgrade info and will continue with the v3 upgrade/
+
+## Manual Option
+1. Wait for Osmosis to reach the upgrade height (xxx)
```
Comment (puneet2019) [inline core-1/upgrades/v3/guide.md:78]:
```suggestion
1. Wait for PersistenceCore to reach the upgrade height (xxx)
```

File: core-1/upgrades/v3/guide.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+2. Look for a panic message, followed by endless peer logs. Stop the daemon
+3. Run the following commands:
+```
+cd $HOME/persistenceCore
+git pull
+git checkout v3.1.0
+make install
+```
+4. Start the osmosis daemon again, watch the upgrade happen, and then continue to hit blocks
```
Comment (puneet2019) [inline core-1/upgrades/v3/guide.md:87]:
```suggestion
4. Start the persistenceCore daemon again, watch the upgrade happen, and then continue to hit blocks
```

File: core-1/upgrades/v2/guide.md (hunk 1 above)
Comment (Anmol1696) [reply core-1/upgrades/v2/guide.md:10]: haha this is the old one, so did not change it, just moved it here from the wiki.

File: test-core-1/upgrades/v3/guide_v3.1.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+cd $HOME/persistenceCore
+git pull
+git checkout v3.1.0
+make build
+cp build/persistenceCore ~/.persistenceCore/cosmovisor/upgrades/v3.1/bin
+```
+Now, at the upgrade height, Cosmovisor will upgrade to the v3.1 binary
+
+## Manual Option
+1. Wait for Osmosis to reach the upgrade height
```
Comment (puneet2019) [inline test-core-1/upgrades/v3/guide_v3.1.md:63]:
```suggestion
1. Wait for PersistenceCore to reach the upgrade height
```

File: test-core-1/upgrades/v3/guide_v3.1.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+2. Look for a panic message, followed by endless peer logs. Stop the daemon
+3. Run the following commands:
+```
+cd $HOME/persistenceCore
+git pull
+git checkout v3.1.0
+make install
+```
+4. Start the osmosis daemon again, watch the upgrade happen, and then continue to hit blocks
```
Comment (puneet2019) [inline test-core-1/upgrades/v3/guide_v3.1.md:72]:
```suggestion
4. Start the persistenceCore daemon again, watch the upgrade happen, and then continue to hit blocks
```

**Review by puneet2019 — APPROVED** [review]

File: core-1/upgrades/v3/guide.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
+# v2 to v3
+
+Persistence v3 gov proposal: xxx
+Countdown: xxx
+Height: xxx
+
```
Comment (Anmol1696) [inline core-1/upgrades/v3/guide.md:6]:
```suggestion
Release: [v3](https://github.com/persistenceOne/persistenceCore/releases/tag/v3.1.1)

```

**Pull request merged**